network/v1/types.go

```diff
@@ -72,6 +72,11 @@ type HostSubnet struct {
 	// EgressIPs is the list of automatic egress IP addresses currently hosted by this node
 	// +optional
 	EgressIPs []string `json:"egressIPs,omitempty" protobuf:"bytes,5,rep,name=egressIPs"`
+	// EgressCIDRs is the list of CIDR ranges available for automatically assigning
```

Comment:
Set by the controller

Reply:
I don't understand why egressIPs would be read only?  Is it because the controller will populate egressIPs automatically?

Reply:
Yes. For backward-compatibility we have to continue letting people manually assign egress IPs (by setting `EgressIPs` while leaving `EgressCIDRs` unset). But if they are setting `EgressCIDRs`, then they can't also set `EgressIPs` themselves, because the controller will just end up overwriting it

Reply:
Please add that to the godoc.

Reply:
updated